bpf/lib/egress_gateway.h

```diff
@@ -76,6 +78,9 @@ bool egress_gw_request_needs_redirect(struct iphdr *ip4, int ct_status,
 	*tunnel_endpoint = egress_gw_policy->gateway_ip;
 	return true;
 #else
+	(void)ip4;
+	(void)ct_status;
+	(void)tunnel_endpoint;
```

Comment:
This is to suppress the warnings about unused parameters, right? Then let's use `__maybe_unused`.